src/tools/perf/api/libperf.h

```diff
@@ -297,23 +315,12 @@ typedef struct ucx_perf_params {
         ucp_perf_datatype_t     recv_datatype;
         size_t                  am_hdr_size; /* UCP Active Message header size
                                                 (not included in message size) */
-        int                     is_daemon_mode;  /* Whether DPU offloading daemon
-                                                    is configured */
-        struct sockaddr_storage dmn_local_addr;  /* IP and port of local daemon,
-                                                    used to offload communication */
-        struct sockaddr_storage dmn_remote_addr; /* IP and port of remote daemon,
-                                                    used to offload communication */
+        ucp_perf_daemon_info_t  dmn_info;    /* DPU offloading daemon info */
```

Comment:
why need  a separate struct?

Reply:
To copy global-params daemon config into per-test params, please see my other comment on this

Reply:
will be reverted